src/pages/MyProfile/MyProfile.test.js

```diff
@@ -150,9 +154,14 @@ describe('Validation', () => {
       history: { push: pushSpy }
     })
     const nameImput = getByLabelText('Name')
-    fireEvent.change(nameImput, { target: { value: 'Alice' } })
-    const emailInput = getByLabelText('Email')
-    fireEvent.change(emailInput, { target: { value: 'alice@example.com' } })
+    await act(async () => {
+      fireEvent.change(nameImput, { target: { value: 'Alice' } })
+      await wait(0)
+    })
+
+    // NOTE: Currently we will not be updating email.
```

Comment:
let's just remove the commented code.

Reply:
I was leaving this code visible, but commented out, as we likely will be needing it in the near future (after our first release)... but I also like the cleanliness without the noise of the comments. We can remove it for now. :+1: